generators/lua/lua_generator.ts

```diff
@@ -18,24 +18,23 @@ import {Names} from '../../core/names.js';
 import type {Workspace} from '../../core/workspace.js';
 import {inputTypes} from '../../core/inputs/input_types.js';
 
-
 /**
  * Order of operation ENUMs.
  * http://www.lua.org/manual/5.3/manual.html#3.4.8
  * @enum {number}
  */
 export enum Order {
-  ATOMIC = 0,    // literals
+  ATOMIC = 0, // literals
   // The next level was not explicit in documentation and inferred by Ellen.
-  HIGH = 1,            // Function calls, tables[]
-  EXPONENTIATION = 2,  // ^
-  UNARY = 3,           // not # - ~
-  MULTIPLICATIVE = 4,  // * / %
-  ADDITIVE = 5,        // + -
-  CONCATENATION = 6,   // ..
-  RELATIONAL = 7,      // < > <=  >= ~= ==
-  AND = 8,             // and
-  OR = 9,              // or
+  HIGH = 1, // Function calls, tables[]
+  EXPONENTIATION = 2, // ^
+  UNARY = 3, // not # - ~
+  MULTIPLICATIVE = 4, // * / %
+  ADDITIVE = 5, // + -
+  CONCATENATION = 6, // ..
+  RELATIONAL = 7, // < > <=  >= ~= ==
+  AND = 8, // and
+  OR = 9, // or
```

Comment:
Please restore the original formatting here and use `// prettier-ignore` to preserve it.